### DISCUSSION TITLE
I wanna join team sir

### DISCUSSION THREAD

**Discussion by SKetU-l** [ROM-builders/temporary#18439] [General]
https://github.com/SKetU-l/android_device_asus_X00TD.git

https://cirrus-ci.com/build/6451501598507008
https://cirrus-ci.com/build/6143766587768832 

https://t.me/SKetUl here's my telegram

Maybe I guess I have fulfilled all requirements if anything left le me know thanks (⁠^⁠^⁠)

edit: am recently working on another device too RMX1801(Realme 2 pro) replying this will be appreciated

**Comment by athizz2005** [comment]
no

**Reply by SKetU-l** [reply]
Bete 💀

**Comment by Apon77** [comment]
seems like your account is inactive

**Reply by SKetU-l** [reply]
Lmao hell no check cirrus am building for RMX1801 these days
